src/components/NewCommentForm.tsx

```diff
@@ -1,6 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Comment, CommentData } from '../types/Comment';
+import { Post } from '../types/Post';
+import { ErrorMessage } from '../types/ErrorMessage';
+import { ShowLoader } from '../types/ShowLoader';
+import { client } from '../utils/fetchClient';
+
+type Props = {
+  activePost: Post | null,
+  onAddComments: (newComment: Comment) => void,
+  setIsProcessing:(string: string) => void,
+  isProcessing: string,
+  setErrorMessage:(message: ErrorMessage) => void
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  activePost,
+  onAddComments,
+  isProcessing,
+  setIsProcessing,
+  setErrorMessage,
+}) => {
+  const defaultComment = {
```

Comment:
Move the defaultComment variable outside the component to not create it on each render 
```
const defaultComment = { ...

export const NewCommentForm...